fastlane/README.md

```diff
@@ -0,0 +1,29 @@
+fastlane documentation
```

Comment:
do we want this?
also, do we need to add some kind of license if we use fastlane?